server/src/ClassVisitor.ts

```diff
@@ -79,21 +79,22 @@ export class ClassVisitor implements YmlToBdlVisitor<void> {
       }
     }
   }
-  getDocumentation(options: FieldContext[]): any {
+  getDocumentation(fieldOptions: FieldContext[]): any {
     try {
-      for (const option of options) {
+      for (const option of fieldOptions) {
         if (option._optionName.text === "documentation") {
-          const documentation = option._optionValues[0].text;
+          let documentation = option._optionValues[0].text;
           if (documentation !== null && documentation !== undefined) {
+            documentation = documentation.replace(/^"+/, "");
+            documentation = documentation.replace(/"+$/, "");
```

Comment:
I might be wrong, but it seems that it can only be `"` or `"""` (and there can be weird newlines and stuff), so I'd advise to use something like…

```
^("|""")\s*
\s*("|""")$
```